python/flyft/state.py

```diff
@@ -51,10 +51,10 @@ def __init__(self, L, shape, origin=0.):
         super().__init__(L, shape, origin)
 
     @property
-    def center(self):
-        if not hasattr(self, '_center'):
-            self._center = np.array([self._self.center(i) for i in range(self.shape)])
-        return self._center
+    def centers(self):
+        if not hasattr(self, '_centers'):
+            self._centers = np.array([self._self.centers(i) for i in range(self.shape)])
```

Comment:
This method is not plural, it will crash if you actually run code with it:
```suggestion
            self._centers = np.array([self._self.center(i) for i in range(self.shape)])
```